lucille-core/src/test/java/com/kmwllc/lucille/indexer/SolrIndexerIntegrationTest.java

```diff
@@ -36,6 +39,14 @@
 @ThreadLeakScope(ThreadLeakScope.Scope.SUITE)
 public class SolrIndexerIntegrationTest extends SolrCloudTestCase {
 
+  /*
```

Comment:
suggest moving this comment to body of testValidateConnection() and/or adding an abbreviated version of this comment to the specific places in the test code where it applies